dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.classpath

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```

Comment:
this .classpath won't work in OpenLiberty.  Check build.example_fat/.classpath for an example

Reply:
updated